tests/console/nftables.pm

```diff
@@ -104,7 +101,7 @@ true');
 
     # check firewall settings and save them to logs
     assert_script_run('echo -e "____ firewall services --zone=public ____\n" >> ' . $logs);
-    assert_script_run("firewall-cmd --list-services --zone=public |& tee -a $logs");
+    assert_script_run("firewall-cmd --list-services --zone=public | grep -e dhcpv6-client |& tee -a $logs");
```

Comment:
All of the calls that you are doing, that send any output to the logs will produce false positives, because that call to `tee` will hide the exit code from the `grep call`

```suggestion
    assert_script_run("firewall-cmd --list-services --zone=public | grep -e dhcpv6-client ");
```

Reply:
well, then I need to change this back again as I added last time and okurz was not agree with it :)

assert_script_run("firewall-cmd --list-services --zone=public |& tee -a $logs");

Reply:
@Zaoliang please use the suggestion I propose. `|& tee -a $logs` will hide any problems.

Reply:
yes, thanks!